spec/models/custom_button_spec.rb

```diff
@@ -354,10 +354,7 @@
             end
 
             expect(CustomButtonEvent.count).to eq(3)
-            expect(CustomButtonEvent.first).to have_attributes(
-              :source      => 'UI',
-              :target_id   => vm.id,
-              :target_type => 'VmOrTemplate',
+            expect(CustomButtonEvent.find_by(:target_id => vm.id, :target_type => "VmOrTemplate", :source => 'UI')).to have_attributes(
```

Comment:
Does this work for these?
```ruby
expect(CustomButtonEvent.find_by(:target => vm, :source => 'UI')).to have_attributes(
```

Reply:
:woman_shrugging: i thought so but nick wanted this so this's what it is now.